README.md

```diff
@@ -13,19 +13,12 @@ This project implements dynamic module loading through `System` exactly to the p
 * Can be used as a [tracing tool](docs/tracing-api.md) for static analysis of modules.
 * Supports IE8+, with IE9+ support for ES6 development without pre-compilation.
 * The minified production loader is under 5KB minified and gzipped, making it suitable for production use, provided that modules are [built into ES5 making them independent of Traceur](docs/production-workflows.md).
+* Supports declaring modules with `<script type="module">`, the precursor of the proposed [`<module>` tag](https://github.com/dherman/module-tag/).
 
 For an overview of build workflows, [see the production guide](docs/production-workflows.md).
 
 For an example of a universal module loader based on this polyfill for loading AMD, CommonJS and globals, see [SystemJS](https://github.com/systemjs/systemjs).
 
-### Documentation
-
-* [Configuring the loader](docs/loader-config.md)
-* [Production workflows](docs/production-workflows.md)
-* [Circular References &amp; Bindings](docs/circular-references-bindings.md)
-* [Extending the loader through loader hooks](docs/loader-extensions.md)
-* [Tracing API](docs/tracing-api.md)
```

Comment:
These links just repeat what's in the previous section (in a more concise form), and it seemed to me that people coming to this file would want to read the getting started section first, and only then maybe want to move on to more details. So put the documentation links at the end